app/src/main/java/org/dhis2/utils/DateUtils.java

```diff
@@ -1153,7 +1155,8 @@ public Boolean isEventExpired(Date eventDate, Date completeDate, EventStatus sta
                 expDate = calendar.getTime();
             }
 
-            expiredBecouseOfPeriod = expDate != null && expDate.before(getCalendar().getTime());
+//            expiredBecouseOfPeriod = expDate != null && expDate.before(getCalendar().getTime());
+            expiredBecouseOfPeriod = expDate != null && expDate.compareTo(getCalendar().getTime())<=0;
```

Comment:
Will this commented code be needed in the near future?

Reply:
Not needed